test/integration.js

```diff
@@ -0,0 +1,7 @@
+import Spark from '../lib/spark.js'
+import { test } from 'zinnia:test'
+
+test('integration', async () => {
+  const spark = new Spark()
+  await spark.nextRetrieval()
+})
```

Comment:
I am unsure if we agreed to fetch retrieval details in this test from Spark API. If we want to, then this test needs an update after https://github.com/filecoin-station/spark-api/pull/21 is deployed.